python/lsst/pipe/drivers/constructCalibs.py

```diff
@@ -228,9 +231,41 @@ def getCcdIdListFromExposures(expRefList, level="sensor", ccdKeys=["ccd"]):
                 ccdLists[name] = []
             ccdLists[name].append(ccdId)
 
+    for ccd in ccdLists:
+        ccdLists[ccd] = sorted(ccdLists[ccd], key=lambda dd: dictToTuple(dd, sorted(dd.keys())))
```

Comment:
Each item is a dict and you are first sorting each item by key, then lexicographically the list by value? Comment might be useful here.